app/controllers/albums

```diff
@@ -0,0 +1,27 @@
+const rp = require('request-promise');
+const typicodePath = 'https://jsonplaceholder.typicode.com';
+
+const options = {
+    uri: '',
+    json: true, // Automatically parses the JSON string in the response
+    resolveWithFullResponse: false
+};
+
+const albums = (req, res, next) => {
```

Comment:
You can do `exports.albums = (req, res, next) => {...}` and avoid having to do `exports.albums = albums;`

Reply:
Also, the function being called `albums` doesn't tell me what is it doing with the albums. Is it getting them? is it setting them? is it deleting them? . I know I can read the code, but if someone can understand a function by just reading the title, then all the better.